lib/url_checker.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'app_config'
+
+class UrlChecker
+
+  def initialize(config)
+    @config = config
+    @govuk_regex = /^#{@config.domains.production}/
+    @parameterised_url_regex = /\?/
```

Comment:
@govuk_regex depends on the incoming value of config, so can't be a constant.  Better I think to keep all three as instance variables, rather than two as constants and one as an instance variable.